README.md

```diff
@@ -201,6 +201,30 @@ message = client.messages.create(
 print(message)
 ```
 
+or if you want to use a profile and region
+
+```py
+import os
+from anthropic import AnthropicBedrock
+
+PROFILE_NAME = os.getenv("PROFILE_NAME")
+REGION_NAME = os.getenv("REGION_NAME")
+
+client = AnthropicBedrock(aws_profile=PROFILE_NAME, aws_region=REGION_NAME)
+
+message = client.messages.create(
+    max_tokens=1024,
+    messages=[
+        {
+            "role": "user",
+            "content": "Hello!",
+        }
+    ],
+    model="anthropic.claude-3-sonnet-20240229-v1:0",
+)
+print(message)
+```
```

Comment:
nit: I think this example is slightly too big, having either a list of supported additional client arguments or a small snippet like this would be best imo:

The bedrock client supports the following arguments for authentication
```py
AnthropicBedrock(
  aws_profile='...',
  aws_region='us-east'
  ... etc
)
```

Reply:
done ✅ @RobertCraigie thank you!